adobe_vipm/flows/fulfillment/shared.py

```diff
@@ -984,6 +1012,18 @@ def __call__(self, client, context, next_step):
         next_step(client, context)
 
 
+class CompleteConfigurationOrder(CompleteOrder):
```

Comment:
I would ask you to have a little bit for flexibility in the `CompleteOrder` step here

1. Modify the complete order step, that if template_name is None - it should call some `get_template_name` function in the `__call__`.
2. This function by default should raise the exception if template_name is None
3. Make template_name as an optional parameter in `CompleteOrder`
4. Here the only thing that you should do is to define the `get_template_name` function

Overriding the whole `__call__` is too much as for me